test/blackbox-tests/test-cases/dir-target-dep.t/run.t

```diff
@@ -15,12 +15,9 @@
 
   $ dune build --root dep
   Entering directory 'dep'
-  File "dune", line 1, characters 0-68:
-  1 | (alias
-  2 |  (name default)
-  3 |  (deps dir)
-  4 |  (action (bash "cat %{deps}/*")))
   Error: No rule found for dir
+  -> required by dir
```

Comment:
Hmm, this seems similar to my above comment about `foo`. Perhaps, we just duplicate some information in error messages? Such duplication doesn't seem nice.

Reply:
Same here